html/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>GWEALTH VISUALS WEBSITE</title>
+</head>
+<body>
+	<header>
+		<h1>Gwealth Visuals</h1>
+		<nav>
+			<ul>
+				<li><a href="#">Portfolio</a></li>
+				<li><a href="#">Blog</a></li>
+				<li><a href="#">Brandings</a></li>
+				<li><a href="#">Showreel</a></li>
+				<li><a href="#">Testimonial</a></li>
+				<li><a href="#">Contact Us</a></li>
+			</ul>
+		</nav>
+	</header>
+	<section>
+		<article>
+			<h2>Branding</h2>
+			<p>Our branding service goes beyond just designing a logo. We help you develop a comprehensive brand strategy that encompasses all aspects of your business, from messaging to visual identity. We work to create a cohesive and memorable brand that resonates with your target audience and helps you stand out in a crowded marketplace.</p>
+		</article>
+		<article>
+			<h2>Printing and Publishing</h2>
+			<p>Our printing and publishing service offers high-quality printing services for a variety of materials, including business cards, brochures, flyers, and more. We also provide publishing services for books, magazines, and other print media. With our state-of-the-art equipment and experienced team, we can produce print materials that showcase your brand in the best possible light.</p>
+		</article>
+		<article>
+			<h2>Video Editing</h2>
+			<p>Our video editing service helps you create engaging and compelling video content that tells your brand's story. We work with you to develop a video concept, storyboard, and script, and then use our editing expertise to bring your vision to life. Whether you need a promotional video, product demo, or social media content, we can help you create video that resonates with your audience and drives results.</p>
+		</article>
+		<aside>
+			<p>Exciting news! GWealth Visuals now offers high-quality shirt printing with our new DTF machine. Contact us today to learn more</p>
+		</aside>
+		<ul>
+			<li>Quality Designs</li>
+			<li> Large Format Printing</li>
+			<li>Professional Corporate Video production</li>
+			<li><a href="https://www.facebook.com/gwealthvisuals/">Click to See More </a></li>
+		</ul>
+		<img src="/image/240.png" alt="my beautiful image">
```

Comment:
Your image won't display, add ".."  at the beginning of the image path.

Using the ".." symbol in file paths allows for more flexibility in organizing files and directories and makes it easier to reference files in different locations within a project.

Reply:
Thank you so much sir
I have corrected the file path mistake

Reply:
sir please kindly note that our instructor asked that I avoid the lint error check because my system couldn't install some files, but i actually did some manual Indentation to codes.